module/data/item/race.mjs

```diff
@@ -0,0 +1,73 @@
+import SystemDataModel from "../abstract.mjs";
+import { AdvancementField, IdentifierField } from "../fields.mjs";
+import ItemDescriptionTemplate from "./templates/item-description.mjs";
+
+/**
+ * Data definition for Race items.
+ * @mixes ItemDescriptionTemplate
+ *
+ * @property {string} identifier       Identifier slug for this race.
+ * @property {object[]} advancement    Advancement objects for this race.
+ * @property {number} darkvision       Darkvision distance.
+ * @property {object} movement
+ * @property {number} movement.burrow
+ * @property {number} movement.climb
+ * @property {number} movement.fly
+ * @property {number} movement.swim
+ * @property {number} movement.walk
+ * @property {string} movement.units   Units used to measure movement.
+ * @property {object} type
+ * @property {string} type.value       Creature type (e.g. "humanoid", "construct").
+ * @property {string} type.subtype     Additional type data (e.g. "elf" for Eladrin)
+ */
+export default class RaceData extends SystemDataModel.mixin(ItemDescriptionTemplate) {
+  /** @inheritdoc */
+  static defineSchema() {
+    return this.mergeSchema(super.defineSchema(), {
```

Comment:
Recommend aliasing:
```js
const fields = foundry.data.fields;
```
at the top of the function so that the rest of the code reads a bit more concisely.